include/login.php

```diff
@@ -0,0 +1,21 @@
+<?php
+function attemptLogin($username, $password){
+    $result=dbQuery('
+        SELECT *
+        FROM users
+        WHERE username = :username
+        AND password = :password',
```

Comment:
Fine for now, but make sure no one actually signs up for real until this is being hashed